### PR TITLE
Publiccode.net codebases content

### PR DESCRIPTION
**Pull request by angelplasma**
This PR contains content for the new HTML codebase overview pages and codebases index page. The words are very close to what is currently live on the [codebases index page](https://publiccode.net/codebases/), [Signalen](https://publiccode.net/codebases/signalen.html) and [OpenZaak](https://publiccode.net/codebases/openzaak.html) pages; this PR represents a significant improvement to the layout.
This PR depends on #110 being merged first.

### CODE REVIEW

File: _layouts/codebase.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
```
Comment (ElenaFdR) [inline _layouts/codebase.html:1]: Should this only be in #110?

**Comment by angelplasma** [comment]
In reference to this error: https://travis-ci.com/github/publiccodenet/publiccode.net/builds/198718071

It's looking for "../assets/style_codebases.css", which won't exist until the Sass CSS (.scss) file has been transpiled by Jekyll into the needed .css file.

**Comment by ElenaFdR** [comment]
Still missing from draft PR:

- [ ] header on codebase overview pages
- [ ] footer on codebase overview pages
- [ ] breadcrumb from codebase overview pages back to codebases/index
- [ ] breadcrumb from codebases/index page back to index.html
- [ ] change 'Our codebases' header on codebase overview pages (and everywhere else this language appears)

**Comment by ElenaFdR** [comment]
@angelplasma VScode also gives me this error: `/favicon.ico' not found

File: codebases/index.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    <h4 class="subtitle"><p>The Foundation for Public Code helps to effectively scale codebases. We do this by supporting ambitious communities with <a href="/codebase-stewardship/">codebase stewardship</a>.</p>
+    <p>Codebases are organized by their <a href="https://about.publiccode.net/activities/codebase-stewardship/lifecycle.html">codebase maturity</a>.</p></h4>
+  </div>
+</div>
+
+<div class="cb-maturity-index-header cb-maturity-s1">
+  <h1 class="title">Stage 1: <strong>Assessment</strong></h1>
+  <p><strong>In this phase, codebase communities and the Foundation for Public Code decide if:</strong></p>
+  <ul>
+    <li>the codebase is a good fit for Foundation stewardship</li>
```
Comment (Ainali) [inline codebases/index.html:18]: Should "Foundation" in "Foundation stewardship" be written in full like "Foundation for Public Code stewardship"? (Applies to the use of "Foundation" four more times on this page.)

File: codebases/index.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+<div class="cb-listings-container">
+  <div class="cb-list-none">There are currently no mature codebases under Foundation stewardship.</div>
+</div>
+
+
+
+<div class="cb-maturity-index-header cb-maturity-s4">
+  <h1 class="title">Stage 4: <strong>Attic</strong></h1>
+  <p><strong>Codebases and communitities that were once Standard compliant but aren't any more are in the attic.</strong></p>
```
Comment (Ainali) [inline codebases/index.html:81]:
```suggestion
  <p><strong>Codebases and communitities that were once Standard compliant but aren't anymore are in the attic.</strong></p>
```

File: codebases/openzaak.html [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>How we're helping</h3>
+    <p>We're helping launch the OpenZaak open source ecosystem by:</p>
+    <ul>
+      <li>Running a <a href="https://github.com/open-zaak/open-zaak-market-consultation/blob/develop/README.md" target="_blank">market consultation</a> to co-create codebase governance and community dynamics, and better understand vendor business models and risk sharing</li>
+      <li>Advising on decentralized procurement</li>
+      <li>Building the community through shared events, a mailing list and marketing materials</li>
```
Comment (Ainali) [inline codebases/openzaak.html:55]: Should we add "setting up governance" before mailing list? We wrote the draft for the governance file and have nudged that forward.

File: codebases/openzaak.html [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section cb-section-links">
+  <div class="cb-detail-info">
+    <h3>Find out more</h3>
+    <ul>
+      <li><a href="https://github.com/open-zaak/open-zaak" target="_blank">Github repo</a></li>
```
Comment (Ainali) [inline codebases/openzaak.html:112]: I guess "repo" works, especially for tech-savvy people, but perhaps repository is easier English?

File: codebases/signalen.html [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+    <h1 class="title">Codebases we work with</h1>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-maturity-detail-header">
+    <div class="cb-maturity-detail-title cb-maturity-s1">
+      <div class="cb-logo"><img src="../photos/codebases/signalen/signalen_logo_small.png" alt="Signalen logo"></div>
```
Comment (Ainali) [inline codebases/signalen.html:17]: In the [guidelines for alt-text](https://www.digg.se/om-oss/nyheter/2021/textalternativ-for-bilder--information-nar-inte-bilden-kan-visas) from the Swedish Agency for digital government they have an explicit example of not using the alt-text "Agency logo" but rather recommend just "Agency" as the text. Not sure if this is a local guideline or universal.

File: codebases/signalen.html [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+        <li>Algorithmic processing by type of complaint, and second layer triaging and prioritisation by humans</li>
+        <li>Automatic updates to complainants about the status of their report</li>
+        <li>Automatic alerts to city staff based on analysis of combined reports</li>
+        <li>Allowing reports to be passed on to other city workers and subcontractors</li>
+      </ul>
+    </div>
+
+    <div class="cb-detail-gallery">
+      <a href="#img1">
+        <img class="lb-thumb" src="../photos/codebases/signalen/signalen_thumb_g01.png" alt="Signalen product thumbnail">
```
Comment (Ainali) [inline codebases/signalen.html:38]:
```suggestion
        <img class="lb-thumb" src="../photos/codebases/signalen/signalen_thumb_g01.png" alt="A web form with a map">
```

File: codebases/signalen.html [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+      </a>
+      <!-- lightbox container hidden with CSS -->
+      <div class="lightbox" id="img1">
+        <a href="#img3" class="lb-btn lb-btn-prev"></a>
+        <a href="#" class="lb-btn lb-btn-close"></a>
+        <a href="#img2"><img src="../photos/codebases/signalen/signalen_g01.png" alt="Signalen product example 1"></a>
```
Comment (Ainali) [inline codebases/signalen.html:44]:
```suggestion
        <a href="#img2"><img src="../photos/codebases/signalen/signalen_g01.png" alt="A map with an address search field"></a>
```

File: codebases/signalen.html [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+        <a href="#img2" class="lb-btn lb-btn-next"></a>
+      </div>
+      <!-- lightbox container hidden with CSS -->
+      <div class="lightbox" id="img2">
+        <a href="#img1" class="lb-btn lb-btn-prev"></a>
+        <a href="#" class="lb-btn lb-btn-close"></a>
+        <a href="#img3"><img src="../photos/codebases/signalen/signalen_g02.png" alt="Signalen product example 2"></a>
```
Comment (Ainali) [inline codebases/signalen.html:51]:
```suggestion
        <a href="#img3"><img src="../photos/codebases/signalen/signalen_g02.png" alt="A map with an address displayed on it"></a>
```

File: codebases/signalen.html [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+        <a href="#img3" class="lb-btn lb-btn-next"></a>
+      </div>
+      <!-- lightbox container hidden with CSS -->
+      <div class="lightbox" id="img3">
+        <a href="#img2" class="lb-btn lb-btn-prev"></a>
+        <a href="#" class="lb-btn lb-btn-close"></a>
+        <a href="#img1"><img src="../photos/codebases/signalen/signalen_g03.png" alt="Signalen product example 3"></a>
```
Comment (Ainali) [inline codebases/signalen.html:58]:
```suggestion
        <a href="#img1"><img src="../photos/codebases/signalen/signalen_g03.png" alt="An overview of an incoming report in the admin interface"></a>
```

File: codebases/signalen.html [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+    </div>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>How we're helping</h3>
+    <p>We're helping Amsterdam and the VNG define a governance structure, including shared feature roadmaps and technical steering.</p>
```
Comment (Ainali) [inline codebases/signalen.html:70]: This is the first time we use VNG on this page, should we explain the abbreviation?

File: codebases/signalen.html [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+    <p>We're also helping make the codebase more conducive to collaboration, including:</p>
+    <ul>
+      <li>Clear branching and gitflow models</li>
+      <li>Automated unit tests and integration testing</li>
+      <li>Engineering guidelines</li>
+      <li>Help prepare the community to empower market parties to offer the solution as SaaS</li>
```
Comment (Ainali) [inline codebases/signalen.html:76]:
```suggestion
      <li>Help prepare the community to empower market parties to offer the solution as Software as a Service (SaaS)</li>
```

File: codebases/signalen.html [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>Opportunity</h3>
+    <dl>
+      <dt>Scaling potential:</dt>
+      <dd>Other Dutch municipalities</dd>
+    </dl>
+    <p>The Association of Dutch Municipalities (VNG) wants to make it 'one click deployable' from their Common Ground appstore.</p>
```
Comment (Ainali) [inline codebases/signalen.html:118]: On line 104 we have the long Dutch form. Should we put them all in one place and then just refer to VNG everywhere else?

File: codebases/signalen.html [hunk 14]
```diff
@@ -0,0 +1,136 @@
...
+<hr>
+
+<div class="cb-section cb-section-links">
+  <div class="cb-detail-info">
+    <h3>Find out more</h3>
+    <ul>
+      <li><a href="https://signalen.org" target="_blank">Signalen website</a></li>
+      <li><a href="https://youtube.com/watch?v=I2Z-mRFt3pg" target="_blank">"Signalen: helping cities manage and prioritize citizens' reports"</a> (video)</li>
+      <li><a href="https://github.com/Amsterdam/signals" target="_blank">Github: backend</a></li>
+      <li><a href="https://github.com/Amsterdam/signals-frontend" target="_blank">Github: fronted</a></li>
```
Comment (Ainali) [inline codebases/signalen.html:132]:
```suggestion
      <li><a href="https://github.com/Amsterdam/signals-frontend" target="_blank">Github: frontend</a></li>
```

File: codebases/openzaak.html [hunk 15]
```diff
@@ -0,0 +1,116 @@
...
+    <div class="cb-detail-info">
+      <h3>What it is</h3>
+      <p>OpenZaak is a case management platform for municipalities that want to deliver citizen-friendly services online.</p>
+      <p>It is a ready-to-use backend solution that acts as a directory of case registration, including metadata describing the case.</p>
+      <p>OpenZaak implements the <a href="https://www.vngrealisatie.nl/producten/api-standaarden-zaakgericht-werken" target="_blank">Dutch national API standards for case management</a>, developed by the Dutch Association of Municipalities (VNG).</p>
+    </div>
+
+    <div class="cb-detail-gallery">
+      <a href="#img1">
+          <img class="lb-thumb" src="../photos/codebases/openzaak/openzaak_thumb_g01.png" alt="OpenZaak product thumbnail">
```
Comment (Ainali) [inline codebases/openzaak.html:33]:
```suggestion
          <img class="lb-thumb" src="../photos/codebases/openzaak/openzaak_thumb_g01.png" alt="Configuration menu of OpenZaak displaying a date picker">
```

File: codebases/openzaak.html [hunk 16]
```diff
@@ -0,0 +1,116 @@
...
+      </a>
+      <!-- lightbox container hidden with CSS -->
+      <div class="lightbox" id="img1">
+        <!--<a href="#img3" class="lb-btn lb-btn-prev"></a>-->
+        <a href="#" class="lb-btn lb-btn-close"></a>
+        <a href="#img1"><img src="../photos/codebases/openzaak/openzaak_g01.png" alt="OpenZaak product example 1"></a>
```
Comment (Ainali) [inline codebases/openzaak.html:39]:
```suggestion
        <a href="#img1"><img src="../photos/codebases/openzaak/openzaak_g01.png" alt="Configuration menu of OpenZaak displaying a date picker"></a>
```

Comment (angelplasma) [reply codebases/openzaak.html:39]: Thank you for writing these much improved alt tags @Ainali !!

File: codebases/signalen.html (hunk 11 above)
Comment (ElenaFdR) [reply codebases/signalen.html]: Thanks, I've fixed this!

File: codebases/signalen.html (hunk 6 above)
Comment (ElenaFdR) [reply codebases/signalen.html:17]: [This website](https://webaim.org/techniques/alttext/#logos) seems to recommend the same.

File: codebases/index.html (hunk 2 above)
Comment (ElenaFdR) [reply codebases/index.html]: I've done this across the website now, except for the job ads.

File: codebases/openzaak.html (hunk 4 above)
Comment (ElenaFdR) [reply codebases/openzaak.html]: Done!

**Review by Ainali — APPROVED** [review]
This is really good! A small improvement that could be done (but it is not a blocker) would be to change the logos from the png versions that are a little blurry to svg files to make them really crisp.

**Comment by ElenaFdR** [comment]
The merge conflict here appears to be related to #132, or would at least be more clear if that were resolved first.

**Pull request merged**